### ISSUE TITLE
Document options

### ISSUE THREAD

**Issue by TimothyGu** [jstransformers/jstransformer-markdown-it#6]
This module provides an options API with plugins and inline. Document it in README.

CC @RobLoach.

**Comment by dillonbheadley** [comment]
I agree. How do i use plugins for markdown-it? Also trying to use plugins within metalsmith-jstransformer. How do I do that?
